lib/flatfile/QubitFlatfileExport.class.php

```diff
@@ -193,6 +193,15 @@ function overrideConfigData(&$config, $mixin)
       }
       else
       {
+        // If config array being overridden is a sequential array,
+        // replace all elements in the array (e.g. isad 'columnNames').
+        // If config array being overridden is an associative array,
+        // override elements are merged so do not use this logic
+        // (e.g. rad 'map').
+        if ($config[$key] === array_values($config[$key]))
+        {
+          $config[$key] = array();
+        }
         $this->overrideConfigData($config[$key], $mixin[$key]);
```

Comment:
Nitpick: blank line after conditional closing brace.

Reply:
Thanks @djjuhasz! I have added the blank line.